styles.css

```diff
@@ -0,0 +1,125 @@
+
+.mobile-header {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0.1px;
+  top: 0.1px;
+  background: #3c3a39;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 48px;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  color: #ff6b00;
+  background: #1c1a19;
+  padding: 120px 24px;
+}
+
+p {
+  position: static;
+  flex: 1;
+  width: 77px;
+  height: 24px;
+  left: 12px;
+  top: 10px;
+  margin: 20px;
```

Comment:
- [ ] To achieve the proper positioning of elements, please use `padding`, `margins` and `display: flex` associated properties like `justify-content` and `align-items`. Having a fixed position with left and top will mess up your responsive design and you're going to have a bigger headache as you keep adding features in your portfolio